network/p2p/p2pnode/observableGossipSub.go

```diff
@@ -1,124 +0,0 @@
-package p2pnode
-
-import (
-	pubsub "github.com/libp2p/go-libp2p-pubsub"
-	"github.com/libp2p/go-libp2p/core/host"
-	"github.com/libp2p/go-libp2p/core/peer"
-	"github.com/libp2p/go-libp2p/core/protocol"
-	"github.com/rs/zerolog"
-
-	"github.com/onflow/flow-go/module"
-)
-
-type ObservableGossipSubRouter struct {
-	router  *pubsub.GossipSubRouter
-	metrics module.GossipSubRouterMetrics
-	logger  zerolog.Logger
-}
-
-func NewObservableGossipSub(h host.Host, metrics module.GossipSubRouterMetrics, logger zerolog.Logger) *ObservableGossipSubRouter {
```

Comment:
This component has been renamed and refactored to `GossipSubControlMessageMetrics`